src/commands/package-and-push.yaml

```
@@ -15,9 +15,11 @@ steps:
   - run: |
       git clone -q --depth=1 --single-branch -b master "git@github.com:giantswarm/$(cat .app_catalog_name).git" .app_catalog
   - run: |
-      helm package --save=false ./helm/<<parameters.chart>> --destination .app_catalog
+      mkdir build && helm package ./helm/<<parameters.chart>> --destination ./build
   - run: |
-      helm repo index --url https://giantswarm.github.com/$(cat .app_catalog_name) .app_catalog
+      helm repo index --url https://giantswarm.github.com/$(cat .app_catalog_name) --merge <<parameters.app_catalog>>/index.yaml ./build
```

@kopiczko can you please review and especially verify that the use of app_catalog parameter is correct on this line? I couldn't find in the docs when should one use
- just dot-syntax
- `<<parameters.param_name>>`
- `$(cat .param_name)`

---

If this is path to the file I think it should be:

```suggestion
      helm repo index --url https://giantswarm.github.com/$(cat .app_catalog_name) --merge .app_catalog/index.yaml ./build
```